db/migrate/20231210061330_add_created_at_to_fts_targets_with_index.rb

```diff
@@ -0,0 +1,34 @@
+# For auto load
+FullTextSearch::Migration
+
+class AddCreatedAtToFtsTargetsWithIndex < ActiveRecord::Migration[6.1]
```

Comment:
Does `[6.1]` work with old (supported) Redmine?

Reply:
> Does [6.1] work with old (supported) Redmine?

fix: [a7714ad](https://github.com/clear-code/redmine_full_text_search/pull/118/commits/a7714ad85cef9f6e17e00bdc627b0d763e76d17a)
Thank you so much. I didn't know that it didn't work and raised the following errors.
```console
rails aborted!
ArgumentError: Unknown migration version "6.1"; expected one of "4.2", "5.0", "5.1", "5.2"
/home/runner/work/redmine_full_text_search/redmine_full_text_search/db/migrate/20231210061330_add_created_at_to_fts_targets_with_index.rb:4:in `<top (required)>'
/home/runner/work/redmine_full_text_search/redmine/lib/redmine/plugin.rb:542:in `up'
/home/runner/work/redmine_full_text_search/redmine/lib/redmine/plugin.rb:574:in `migrate_plugin'
/home/runner/work/redmine_full_text_search/redmine/lib/redmine/plugin.rb:515:in `migrate'
/home/runner/work/redmine_full_text_search/redmine/lib/redmine/plugin.rb:529:in `block in migrate'
/home/runner/work/redmine_full_text_search/redmine/lib/redmine/plugin.rb:528:in `each'
/home/runner/work/redmine_full_text_search/redmine/lib/redmine/plugin.rb:528:in `migrate'
/home/runner/work/redmine_full_text_search/redmine/lib/tasks/redmine.rake:135:in `block (3 levels) in <top (required)>'
bin/rails:4:in `require'
bin/rails:4:in `<main>'
```